core/objects.go

```diff
@@ -361,7 +365,7 @@ type Authorization struct {
 	// in process, these are challenges to be fulfilled; for
 	// final authorizations, they describe the evidence that
 	// the server used in support of granting the authorization.
-	Challenges []Challenge `json:"challenges,omitempty" db:"challenges"`
+	Challenges []Challenge `json:"challenges,omitempty" db:"-"`
```

Comment:
Without this we get a `Challenges varchar(255)` field in the schema (although when `CreateTablesIfNotExists` dies this can disappear).